foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/BaseRecordRelationDecorator.java

```diff
@@ -71,4 +84,26 @@ public void validate() {
     public boolean isOneToManyRelation() {
         return wrapped.getMultiplicity().equals(Multiplicity.ONE_TO_MANY);
     }
+
+    /**
+     * Returns the relation name formatted to uppercase with underscores.
+     *
+     * @return name formatted to uppercase with underscores
+     */
+    public String getUpperSnakecaseName() {
+        return PipelineUtils.deriveUpperUnderscoreNameFromUpperCamelName(getName());
+    }
+
+    /**
+     * Returns the relation name, capitalized.
+     *
+     * @return capitalized name
+     */
+    public String getCapitalizedName() {
+        return StringUtils.capitalize(getName());
+    }
+
+    public String getUncapitalizeName() {
```

Comment:
A: very nit, adding comments and past tense getuncaptializedName.
```suggestion
    /**
     * Returns the relation name, uncapitalized. 
     *
     * @return uncapitalized name
     */
    public String getUncapitalizedName() {
```